src/routes/steps.routes.js

```diff
@@ -0,0 +1,24 @@
+const router = require("express").Router();
+const controller = require("../controllers/steps.controller");
+const methodNotAllowed = require("../errors/methodNotAllowed");
+
+router
```

Comment:
the course and step routes should be authenticated

Reply:
Okay, I will make those chaanges.

Reply:
Made the requested changes in the latest commit "add authenticateUser middleware to routes" - see below